docs/content/sips/017-service-chaining.md

```diff
@@ -61,9 +59,11 @@ To avoid spoofing, Spin must strip any `Host: *.spin.internal` headers from rout
 
 ## Permissions
 
-The developer must enable chaining to a component by adding `http://<component-name>.spin.internal`, or `http://*.spin.internal` to allow any destination component, to the list of allowed outbound hosts.  (We cannot use the application flag as permission because not all components may be allowed to make self-requests.)  We could and arguably should consider that any component with `self` permission also has chaining permission: if a component is allowed to make requests to other parts of the application, who cares if that's routed or chained?  However, the idea of private endpoints complicates this a bit - is there a situation in which a component should be allowed to access public routes but not private endpoints?
+The developer must enable chaining to a component by adding `http://<component-name>.spin.internal`, or `http://*.spin.internal` to allow any destination component, to the list of allowed outbound hosts.  (We cannot use the application flag as permission because not all components may be allowed to make self-requests.)
 
-**DECISION REQUIRED:** Should `allowed_outbound_hosts` including _either_ of `self` or `*.spin.internal` be enough to grant permission for self-requests using chaining?  What about hypothetical private endpoints?
+We could and arguably should consider that any component with `self` permission also has chaining permission: if a component is allowed to make requests to other parts of the application, who cares if that's routed or chained?  However, the idea of private endpoints complicates this a bit - is there a situation in which a component should be allowed to access public routes but not private endpoints?
+
+**The preferred user experience is that the developer must list the `<component-name>.spin.internal` host.**
```

Comment:
What about `http://*` or `*://*:*`? On one hand it would be quite odd for these to _not_ match any specific URL, but you also don't want to accidentally permit internal access when you just wanted a proxy or what have you.

Reply:
Yes, wildcards would still match... sorry, I didn't say that explicitly.